components/bridge/ChainPicker.tsx

```diff
@@ -2,38 +2,49 @@ import React, { useState, useEffect } from "react";
 import { Chain, SupportedChain } from "../../types";
 
 import styles from "../../styles/components/bridge/chainpicker.module.css";
-import { ETH_LOGO } from "../../utils/bridge/helpers";
-
-const CHAINS: Chain[] = [
-	{
-		name: "Cennznet",
-		logo: "images/cennz.svg",
-	},
-	{
-		name: "Ethereum",
-		logo: ETH_LOGO,
-	},
-];
+import { CHAINS } from "../../utils/bridge/helpers";
 
 const ChainPicker: React.FC<{
 	setChain: Function;
+	setOppositeChain: Function;
 	initialChain: SupportedChain;
+	forceChain: SupportedChain;
 	topText?: string;
-}> = ({ setChain, initialChain, topText }) => {
-	const [chains, setChains] = useState<Chain[]>(CHAINS);
+}> = ({ setChain, initialChain, setOppositeChain, forceChain, topText }) => {
+	const [chains, _] = useState<Chain[]>(CHAINS);
 	const [chainDropDownActive, setChainDropDownActive] =
 		useState<boolean>(false);
 	const [selectedChainIdx, setSelectedChainIdx] = useState<number>(0);
+	const [initialRenderComplete, setInitialRenderComplete] =
```

Comment:
Why do we need to know about `initialRenderComplete` state?

Reply:
I put this in here as a safeguard because I was getting an infinite event loop on initial render